lib/bolt/outputter/human.rb

```diff
@@ -308,7 +308,7 @@ def format_table(results, padding_left = 0, padding_right = 3)
         )
       end
 
-      def print_tasks(tasks, modulepath)
+      def print_tasks(tasks:, modulepath:)
```

Comment:
Yessssssss

Reply:
Also just checked this is only ever called from Bolt https://github.com/search?q=org%3Apuppetlabs+print_tasks&type=code :+1: